Makefile.doc

```diff
@@ -250,7 +250,7 @@ $(DOC_GRAM): $(DOC_GRAMCMO) coqpp/coqpp_parser.mli coqpp/coqpp_parser.ml doc/too
 PLUGIN_MLGS := $(wildcard plugins/*/*.mlg)
 OMITTED_PLUGIN_MLGS := plugins/ssr/ssrparser.mlg plugins/ssr/ssrvernac.mlg plugins/ssrmatching/g_ssrmatching.mlg \
 	plugins/ssrsearch/g_search.mlg
-DOC_MLGS := $(wildcard */*.mlg) $(sort $(filter-out $(OMITTED_PLUGIN_MLGS), $(PLUGIN_MLGS)))
+DOC_MLGS := $(wildcard */*.mlg) $(sort $(filter-out $(OMITTED_PLUGIN_MLGS), $(PLUGIN_MLGS))) user-contrib/Ltac2/*.mlg
```

Comment:
Why not use `wildcard`?

Reply:
`make` is passing the single matching file, so it works.  I remember adding wildcard here some months ago, but I don't recall exactly when/why it's needed.